templates/tfengine/components/cicd/env/triggers/main.tf

```diff
@@ -80,18 +79,18 @@ resource "google_cloudbuild_trigger" "plan_scheduled" {
   }
   {{- end}}
 
-  filename = "${local.terraform_root_prefix}cicd/configs/tf-plan.yaml"
+  filename = "${local.terraform_root_prefix}cicd/configs/tf-${var.command}.yaml"
 
   substitutions = {
     _TERRAFORM_ROOT = local.terraform_root
     _MANAGED_DIRS = var.managed_dirs
   }
 }
 
-resource "google_cloud_scheduler_job" "plan_scheduler" {
-  count     = (!var.skip && var.run_on_schedule != "") ? 1 : 0
+resource "google_cloud_scheduler_job" "scheduler" {
+  count     = (var.scheduled.skip || var.scheduled.disabled) ? 0 : 1
```

Comment:
will this logic cause issues?

For validate and plan triggers, var.scheduled.disabled is always set to true.